crates/cli/src/subcommands/sql.rs

```diff
@@ -32,13 +32,7 @@ pub fn cli() -> clap::Command {
                  .long("interactive")
                  .action(ArgAction::SetTrue)
                  .conflicts_with("query")
-                 .help("Runs an interactive command prompt for `SQL` expressions"),)
-        .group(
-            ArgGroup::new("mode")
-                .args(["interactive","query"])
-                .multiple(false)
-                .required(true)
-        )
```

Comment:
this was redundant since both the args use `.conflicts_with`. However, putting them in a group caused `clap` to put the **anonymous** query arg before `database` in the help text, even though it was actually the second anonymous arg.